practicals/Recommender_Systems.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>Seems to me that this is the first time you're actually discussing what you can use the embeddings for. I might have missed something above, but if not, then when you're discussing matrix factorisation it would be good to briefly mention WHY it's useful to obtain embeddings, and what they can be used for downstream.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>